File: java/URI_1049.java
package URI;
import java.util.Scanner;

public class URI_1049 {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		String a = sc.next();
		String b = sc.next();
		String c = sc.next();
		
		if(a.equals("vertebrado")) {
			if(b.equals("ave")) {
				if(c.equals("carnivoro")) {
					System.out.println("aguia");
				}
				else {
					System.out.println("pomba");
				}
			}
			else {
				if(c.equals("onivoro")) {
					System.out.println("homem");
				}
				else {
					System.out.println("vaca");
				}
			}			
		}
		else {
			if(b.equals("inseto")) {
				if(c.equals("hematofago")) {
					System.out.println("pulga");
				}
				else {
					System.out.println("lagarta");
				}
			}
			else {
				if(c.equals("hematofago")) {
					System.out.println("sanguessuga");
				}
				else {
					System.out.println("minhoca");
				}
			}
		}
		
		sc.close();
	}

}

File: kotlin/URI_1049.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val a = sc.next()
    val b = sc.next()
    val c = sc.next()

    val animal = if (a == "vertebrado") {
        if (b == "ave") {
            if (c == "carnivoro") {
               "aguia"
            } else {
                "pomba"
            }
        } else {
            if (c == "onivoro") {
                "homem"
            } else {
                "vaca"
            }
        }
    } else {
        if (b == "inseto") {
            if (c == "hematofago") {
               "pulga"
            } else {
              "lagarta"
            }
        } else {
            if (c == "hematofago") {
               "sanguessuga"
            } else {
               "minhoca"
            }
        }
    }
    println(animal)
}
